CHANGELOG.unreleased.md

```diff
@@ -11,6 +11,7 @@ For upgrade instructions, please check the [migration guide](MIGRATIONS.released
 [Commits](https://github.com/scalableminds/webknossos/compare/23.03.1...HEAD)
 
 ### Added
+- WEBKNOSSOS screenshots now include a logo in the lower left corner. [#6910](https://github.com/scalableminds/webknossos/pull/6910)
```

Comment:
If you add the possibility to disable the logo, please add a sentence here on who can disable it.